pkg/interface/publish/src/js/components/lib/join.js

```diff
@@ -1,13 +1,125 @@
 import React, { Component } from 'react'
+import classnames from 'classnames';
+import { Route, Link } from 'react-router-dom';
+import urbitOb from 'urbit-ob';
 
 //TODO textarea + join button to make an api call
 export class JoinScreen extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      book: '/',
+      error: false
+    };
+
+    this.bookChange = this.bookChange.bind(this);
+  }
+
+  componentDidUpdate(prevProps, prevState) {
+      const { props, state } = this;
+      if (state.book in props.notebooks) {
```

Comment:
props.notebooks looks like this:

```
{
  "notebooks": {
    "~zod": {
      "test": {
        "title": "test",
        "about": "test",
        "num-unread": 0,
        "num-notes": 1,
        "date-created": 1580765730419
      }
    }
  }
}
```

Because `state.book` is checking entire notebook titles against the top level properties of the object, not the notebooks nested in those properties, it's checking against notebook owners.

So, basically what this line does, given that notebooks example above, is that if I type `~zod` it boots me back to the home screen.